src/clims/services/workflow/camunda/client.py

```diff
@@ -56,17 +57,17 @@ def start_workflow(self, workflow, item):
         else:
             raise UnexpectedHttpResponse(json["message"], response.status_code)
 
-    def get_tasks_by_ids(self, task_ids):
+    def get_work_units_by_ids(self, task_ids):
         # TODO-perf: We must be able to fetch this in a batch call. Look into setting up
         # https://github.com/camunda/camunda-bpm-graphql (preferably) or direct DB access to resolve
         # this, as it's expected that these will be many calls
         # TODO: paging
-        from clims.services.workflow import ProcessTask
+        raise NotImplementedError()
```

Comment:
This is just here because it's still a WIP branch, part of the refactoring I haven't finished yet. I will fix this when I remove the draft label.

I agree about not keeping code around and I only keep it around if it's to be filled in an upcoming pull request. Git is great when it comes to finding old stuff. I recommend checking out `git log -Ssome_function_name` which shows every change to that function name. So when one removes the function, it's easy to find it again with this command. Another option is to just walk through all revisions of the file from the beginning. I do that via extra tooling (vim plugin), but I think pycharm has the same.